app/scripts/modules/amazon/src/securityGroup/configure/EditSecurityGroupCtrl.js

```diff
@@ -90,6 +90,7 @@ module.exports = angular
       let command = {
         credentials: group.accountName,
         name: group.name,
```

Comment:
can't we remove `name` from the payload?

Reply:
unfortunately no because orca needs it

Reply:
I'm not sure I understand, I thought you submitted a PR against orca to switch from `name` to `securityGroupName`?

Reply:
I take it back. We could remove it... But that would make logs hard to read.
Basically, what happens, `orca` assumes the `name` field is for it and removes it for context but uses it for all human readable references to this task.
But because it removes it the task doesn't get it and hence can't execute. This is, as we discovered only an issue in execution.

But you're right since the UI only calls this on the `/tasks` endpoint we should be able to safely remove it. Let me try

Reply:
after some more investigation, this is ok to remove, thanks!